src/types.ts

```diff
@@ -111,16 +143,19 @@ export enum PlaceBidStatus {
 export interface NewBidParameters {
   tokenId: string;
   bidAmount: string; // in wei
+  bidToken: string;
   startBlock?: string;
   expireBlock?: string;
 }
 
 export interface BuyNowParams {
   amount: string;
   tokenId: string;
+  paymentToken?: string;
 }
 
 export interface Listing {
   price: ethers.BigNumber;
   holder: string;
+  paymentToken?: string;
```

Comment:
This should be non-nullable. If the `paymentToken` is the incorrect token then we default to WILD.